editor/components/inserter/group.js

```diff
@@ -77,7 +77,7 @@ export default class InserterGroup extends Component {
 		return (
 			<NavigableMenu
 				className="editor-inserter__category-blocks"
-				orientation="vertical"
+				orientation="undefined"
```

Comment:
Minor: I know this is the equivalent of the aria value, but it's weird. I'd prefer something like "all" personally